packages/amplify-util-uibuilder/src/clients/amplify-studio-client.ts

```diff
@@ -91,71 +140,103 @@ export default class AmplifyStudioClient {
       } while (nextToken);
       printer.debug(JSON.stringify(uiBuilderComponents, null, 2));
       return { entities: uiBuilderComponents };
-    } catch (e) {
-      printer.debug(e);
-      throw e;
+    } catch (err) {
+      throw new Error(`Failed to list components: ${err.message}`);
```

Comment:
use the new AmplifyError or AmplifyFault class,
this reminds me that we should add a lint for this